src/qml/FeatureForm.qml

```diff
@@ -340,6 +335,11 @@ Page {
   }
 
   function buffer() {
+      if( !model.constraintsValid ) {
+          displayToast( qsTr("Constraints not valid - cannot buffer") )
```

Comment:
Yes I did :slightly_smiling_face:

Reply:
I replaced them. I used single in the new created file and I replaced them all in this file. I don't replace them everywhere I see them because the review would getting more and more complex. So I can do it probably one time in a single PR...